contracts/contracts/core/ProviderRegistry.sol

```diff
@@ -98,6 +98,7 @@ contract ProviderRegistry is
      * @param residualBidPercentAfterDecay The residual bid percent after decay.
      */
     function slash(
+        bytes32 commitmentDigest,
```

Comment:
Isn't this already emitted in the `SlashInitiated` event, during the calling function